docs/contributing/04-prow-workflow.md

```diff
@@ -0,0 +1,91 @@
+
+## OWNERS_ALIASES
+
+The OWNERS_ALIASES file contains a list of aliases that can be used in the OWNERS files, instead of the GitHub usernames.
+It is useful when the same set of people is responsible for bigger amount of directories in the repo.
+
+The example structure of this file is as follows:
+```yaml
+aliases:
+  owners-group:
+    - githubUser1
+    - githubUser2
+    - githubUserN
+```
+
+For more information, refer to the official [Kubernetes documentation](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#owners_aliases).
+
+## Code review process
+
+Since in Kyma Prow is responsible for ensuring the review process, our flow is the same as in the official [Kubernetes documentation](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#the-code-review-process).
+
+## Commands
+
+Prow provides GitHub automations and ChatOps using `/foo` styled slash-commands. You can use these command by creating a comment under the GitHub Issue or Pull Request.
+This flow can be extended by various built-in plugins and can be further extended with external integrations with Prow API that it provides. 
+As part of the workflow every developer needs to know a basic set of commands that they can use for managing Issues or Pull Requests.
+
+Prow plugins implement huge set of command everyone, or specific people can use.
+
+Below there is a basic set of commands that need to be known by every person that wants to contribute to Kyma:
+
+|command|example|description|used by|plugin|
+|---|---|---|---|---|
+|`/close`|`/close`|Close an issue or PR.|authors or members of organisation|lifecycle|
+|`/reopen`|`/reopen`|Reopen an issue or PR.|authors or members of organisation|lifecycle|
+|`/[remove-]lifecycle [stale/frozen/rotten/active]`| `/lifecycle frozen`, `/remove-lifecycle stale`|Command used to operate on lifecycle labels respected by [kyma-stale-bot](https://github.com/apps/kyma-stale-bot).|anyone|lifecycle|
+|`/[un][remove-]hold [cancel]`|`/hold`, `/unhold`, `/remove-hold` `/hold cancel`|Adds or removes the `do-not-merge/hold` Label which is used to indicate that the PR should not be automatically merged.|anyone|hold|
+|`/auto-cc`|`/auto-cc`|Request reviews based on the OWNERS files if those were not assigned.|anyone|blunderbuss|
+|`/[un]cc [@username]`|`/cc @Ressetkk`, `/uncc`|Request review from the specific person, or yourself.|anyone|assign|
+|`/[un]assign [@username]`|`/assign` `/unassign @Ressetkk`|(Un)assign a person from issue or PR.|anyone|assign|
+|`/[remove-](area,kind,priority,label,language) [name]`|`/area prow`, `/label Epic`, `/kind bug` `/remove-area ci`|Applies or removes a label from one of the recognized types of labels.|Anyone can trigger this command on issues and PRs. `triage/accepted` can only be added by org members. Restricted labels are only able to be added by teams and users in their configuration.|label|
```

Comment:
```suggestion
|`/[remove-](area,kind,priority,label,language) [name]`|`/area prow`, `/label Epic`, `/kind bug` `/remove-area ci`|Applies or removes a label from one of the recognized types of labels.| Anyone can trigger this command on issues and PRs. `triage/accepted` can only be added by organization members. Only teams can add the restricted labels.|label|
```

Reply:
`... and users in their configuration` - what does this part mean?
Also, what are the restricted labels?

Reply:
Based on the Prow documentation
```go
// RestrictedLabels allows to configure labels that can only be modified
	// by users that belong to at least one of the configured teams. The key
	// defines to which repos this applies and can be `*` for global, an org
	// or a repo in org/repo notation.
	RestrictedLabels map[string][]RestrictedLabel `json:"restricted_labels,omitempty"`
```